app/controllers/concerns/filter_by_owner.rb

```diff
@@ -8,7 +8,7 @@ module FilterByOwner
   def add_owner_ids_to_filter_param!
     owner_filter = params.delete(:owner).try(:split, ',')
     unless owner_filter.blank?
-      groups = UserGroup.where(name: owner_filter)
+      groups = UserGroup.where('lower(name) = ?', owner_filter.map(&:downcase))
```

Comment:
This used to do a `WHERE name IN ('a', 'b', 'c')` instead of comparison, does equality comparison work for this too in Postgresql?